lib/active_model_serializers/railtie.rb

```diff
@@ -1,7 +1,17 @@
 require 'rails/railtie'
 
-module ActiveModel
+module ActiveModelSerializers
   class Railtie < Rails::Railtie
+    initializer 'active_model_serializers.action_controller' do
+      ActiveSupport.on_load(:action_controller) do
+        ActiveSupport.run_load_hooks(:active_model_serializers, ActiveModelSerializers)
+        include ::ActionController::Serialization
+        ActionDispatch::Reloader.to_prepare do
+          ActiveModel::Serializer.serializers_cache.clear
+        end
+      end
+    end
```

Comment:
Having the `include ::ActionController::Serialization` in an initializer is good for being able to conditionally disable it based on whatever gets decided from #1295. And having the `on_load(:action_controller)` is good for lazy-loading in case `ActionController::Base` is never loaded. Those both seem good.

The `to_prepare` block does not need to be in the initializer. It could be outside the initializers on the Railtie itself with just...
```ruby
  config.to_prepare do
    ActiveModel::Serializer.serializers_cache.clear
  end
```

Using an initializer, and a load hook (`:action_controller`), and then running another load hook (`:active_model_serializers`) to set the logger seems like overly complicated. Depending on load ordering, someone could have an initializer that explicitly sets `ActiveModelSerializers.logger` but then gets overwritten. 
Is there any other reason to have the `:active_model_serializers` lazy-load hook? Could that logic just get folded in where the `run_load_hooks` call is? Would you ever expect someone else to hook into that? And if so, would it only be after :action_controller has been loaded?

I'm still poking around a bit to see if I've got the order right on some of these.